bin/MadGraph5_aMCatNLO/cards/production/2017/14TeV/ZprimeTo2ChiTo2L/ZprimeTo2ChiTo2L_proc_card.dat

```diff
@@ -0,0 +1,40 @@
+#************************************************************
+#*                     MadGraph5_aMC@NLO                    *
+#*                                                          *
+#*                *                       *                 *
+#*                  *        * *        *                   *
+#*                    * * * * 5 * * * *                     *
+#*                  *        * *        *                   *
+#*                *                       *                 *
+#*                                                          *
+#*                                                          *
+#*         VERSION 2.5.5                 2017-05-26         *
+#*                                                          *
+#*    The MadGraph5_aMC@NLO Development Team - Find us at   *
+#*    https://server06.fynu.ucl.ac.be/projects/madgraph     *
+#*                                                          *
+#************************************************************
+#*                                                          *
+#*               Command File for MadGraph5_aMC@NLO         *
+#*                                                          *
+#*     run as ./bin/mg5_aMC  filename                       *
+#*                                                          *
+#************************************************************
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set loop_color_flows False
+set gauge unitary
+set complex_mass_scheme False
+set max_npoint_for_channel 0
+import model sm
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
+define l+ = e+ mu+
+define l- = e- mu-
+define vl = ve vm vt
+define vl~ = ve~ vm~ vt~
+import model UMSSM -modelname
+generate p p > zp > c1 c1bar, (c1 > wm n1, (wm > l- vl~)), (c1bar > wm\
```

Comment:
should the l include a tau ? 
is the model 4SF of 5FS?

Reply:
The model is 5FS, I checked

Reply:
If you are doing analysis for onshell zp, it is better to split the ME calculations like "p p > zp, zp > c1 c1bar". It gives you some unhealthy behaviors [1]

[1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/MGSyntaxPitfall#Avoid_using_s_in_series_without

Reply:
> should the l include a tau ?

we don't need tau. I can remove vt for sure.

Reply:
> If you are doing analysis for onshell zp, it is better to split the ME calculations like "p p > zp, zp > c1 c1bar". It gives you some unhealthy behaviors [1]
> 
> [1] https://twiki.cern.ch/twiki/bin/viewauth/CMS/MGSyntaxPitfall#Avoid_using_s_in_series_without

It's onshell decay. Ok, I'll modify the syntax, thanks for the twiki

Reply:
what about taus that decay to an e or a mu in the end ?

Reply:
considering final lepton pt cuts, BR, etc, there seems not much sensitivity gain from tau contribution. though let me talk with my analysis team